doc/source/data/key-concepts.rst

```diff
@@ -75,12 +75,13 @@ In the common case, each read task produces a single output block. Read tasks ma
 
   Block splitting is off by default. See the :ref:`performance section <data_performance_tips>` on how to enable block splitting (beta).
```

Comment:
Shall we give *block* a more prominent position? I know it's described in Dataset concept, but we do mention it all the places in documentation. We can give it a spot and make it referable.

Reply:
Agreed on this; could I do this in a follow-up PR so we can get these P0s in?